submarine-commons/commons-utils/src/main/java/org/apache/submarine/commons/utils/SubmarineConfiguration.java

```diff
@@ -484,7 +523,10 @@ public void updateConfiguration(String name, String value) {
         "workbench.websocket.max.text.message.size", "1024000"),
     WORKBENCH_WEB_WAR("workbench.web.war", "submarine-workbench/workbench-web/dist"),
     SUBMARINE_RUNTIME_CLASS("submarine.runtime.class",
-        "org.apache.submarine.server.submitter.yarn.YarnRuntimeFactory");
+        "org.apache.submarine.server.submitter.yarn.YarnRuntimeFactory"),
+    SUBMARINE_SUBMITTERS("submarine.submitters", ""),
+    SUBMARINE_SUBMITTERS_ENTRY("submarine.submitters.*.class", ""),
+    SUBMARINE_SUBMITTERS_CLASSPATH("submarine.submitters.*.classpath", "");
```

Comment:
```
    SUBMARINE_SUBMITTERS_ENTRY("submarine.submitters.*.class", ""),
    SUBMARINE_SUBMITTERS_CLASSPATH("submarine.submitters.*.classpath", "");
```
How to configure these 2 variables？
All PR submissions are finished, do you add any more Introduction documents?